google-cloud-spanner/lib/google/cloud/spanner/data.rb

```diff
@@ -140,18 +149,25 @@ def to_a
 
         ##
         # Returns the names or indexes and corresponding values of the data as a
-        # hash. Do not use this method if the data has more than one member with
-        # the same name.
+        # hash. This method will raise or lose values if the data has more than
+        # one member with the same name.
```

Comment:
Maybe: "This method will raise DuplicateNameError if the data has more than one member with the same name, unless `skip_dup_check` is set to `true`, in which case it will lose values."

Reply:
Updated.